HomeWork-7/News/src/main/java/com/example/news/NewsApplication.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
+@EnableScheduling
```

Comment:
Why did you use this annotation?

Reply:
This was for a test "scheduling", forgot to delete it. Deleted